torch/_inductor/utils.py

```diff
@@ -48,6 +48,15 @@ def has_triton():
         return False
 
 
+@functools.lru_cache(None)
+def has_mps():
+    try:
+        torch.randn(1).to("mps")
```

Comment:
Same change which Alban proposed above:
`torch.randn(1, device="mps")`

Reply:
Ho we shouldn't need that call at all haha. Just use torch.backends.mps.is_available() !